CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Change Log
 
 ## [Unreleased]
+### Fixed
+- Record Deletions - Handle the record deletion on models having a composite primary key. 
```

Comment:
We need at least 2 new tests with this fix:
- Record deletion on a simple model (to test regression on this change),
- Record deletion on a model with a composite primary key (to validate the fix you made)